linkedin_api/linkedin.py

```
@@ -712,12 +733,15 @@ def get_conversation(self, conversation_urn_id):
 
         return res.json()
 
-    def send_message(self, conversation_urn_id=None, recipients=[], message_body=None):
+    def send_message(self, conversation_urn_id=None, recipients=None, message_body=None):
         """
         Send a message to a given conversation. If error, return true.
 
         Recipients: List of profile urn id's
         """
+        if not recipients:
+            recipients = []
+
         params = {"action": "create"}
 
         if not (conversation_urn_id or recipients) and not message_body:
```

Should we raise an error if there are no recipients ?